tfjs-backend-webgpu/src/shader_preprocessor_wgsl.ts

```diff
@@ -40,7 +40,7 @@ function mapToTypesWgsl(type: DataType, isVec4: boolean): DataTypeWGSL|
   } else if (type === 'int32') {
     return isVec4 ? 'vec4<i32>' : 'i32';
   } else if (type === 'bool') {
-    return isVec4 ? 'vec4<bool>' : 'bool';
+    return isVec4 ? 'vec4<i32>' : 'i32';
```

Comment:
Why we need to change it from `bool` to `i32`?

Reply:
'vec4<bool>' cannot be used in storage class 'storage' as it is non-host-shareable